build/docker/services/base/update-secrets

```diff
@@ -34,4 +31,37 @@ if [[ -f /root/secrets/token ]]; then
 fi
 
 
+
+# Copy mounted secrets as follows:
+
+# /etc/condor/tokens-orig.d/*                  -> /etc/condor/tokens.d
+# /etc/condor/passwords-orig.d/*               -> /etc/condor/passwords.d
+# /etc/grid-security-orig.d/host{cert,key}.pem -> /etc/grid-security
+# Only files get copied.
+# XXX This does not delete any files from the destination, only overwrites existing ones.
+
+copy_secrets_in_dir () {
+    local from="$1"
+    local to="$2"
+    local owner="$3"
+    shopt -s nullglob
+    if [[ -d $from ]]; then
+        for file in "$from"/*; do
+            if [[ -f $file ]]; then
+                install -o "$owner" -g "$owner" -m 0600 "$file" "$to"/"$(basename "$file")"
+            fi
+        done
+    fi
+    shopt -u nullglob
+}
+
+copy_secrets_in_dir /etc/condor/tokens-orig.d     /etc/condor/tokens.d     condor
```

Comment:
The tokens.d directory is owned by root.
```suggestion
copy_secrets_in_dir /etc/condor/tokens-orig.d     /etc/condor/tokens.d     root
```

Reply:
Fixed by a77531e